docs/admin/access.rst

```diff
@@ -136,13 +139,13 @@ Users, roles, groups and permissions
 The authentication models consist of several objects:
 
 `Permission`
-    Individual permissions defined by Weblate. You can not assign individual
-    permissions, this can only be done through assignment of roles.
+    Individual permissions defined by Weblate. Permissions can not be
```

Comment:
```suggestion
    Permissions can not be
```
Would it be OK to remove this? (On the assumption I go the latter part right)

Reply:
no, let’s keep it